webclient/js/elems/asms.js

```diff
@@ -723,7 +731,7 @@ function buildAssessmentElements() {
                     $("#"+f.name).find("#AssessmentInfo").addClass("hidden");
 
                     // ENABLE ALL INPUTS IF ALL OF THOSE HAVE BEEN DISABLED FOR REVERSED PREVIOUSLY
-                    $("#"+f.name).find('input,button').not('input[name=BUD]').prop("disabled", false);
+                    // $("#"+f.name).find('input,button').not('input[name=BUD]').prop("disabled", false);
```

Comment:
same here! Please, check that BUD field is disabled when you load any assessment that is not reversed!